src/components/form/form.tsx

```diff
@@ -161,6 +161,7 @@ export class Form {
             React.createElement(
```

Comment:
This looks simple enough. What remains to be done?

I assume we'll want to test it thoroughly, to make sure we haven't lost any validation functionality we didn't want to lose.

Reply:
Yes, that's probably my greatest concern as well. Perhaps we should also assess whether we still need extra validation outside of rjsf (just making sure that requirement still stands today).